OracleSOA/dockerfiles/12.2.1.2/Dockerfile.quickstart

```diff
@@ -0,0 +1,40 @@
+# Pull base image
+# ---------------
+FROM oracle/serverjre:8
```

Comment:
Can you review the new structure for the WebLogic `Dockerfile` and try and re-order this `Dockerfile` to match? It looks like there is a lot of similarity between the two and layer re-use can be increased if we can match directives. 

It will also allow future SOA releases to re-use layers.